frontend/src/components/pages/CampCreation/index.tsx

```diff
@@ -164,7 +262,8 @@ const CampCreationPage = (): React.ReactElement => {
         isScheduleSessionsFilled={isScheduleSessionsFilled}
         isRegistrationFormFilled={isRegistrationFormFilled}
       />
-      <Box w="100%" flex="1" mt={[0, "0 !important"]}>
+
+      <Box w="100%" flex="1" mt={[0, "0 !important"]} paddingLeft="200px">
```

Comment:
Curious as to why you added more padding to the left here. Would this impact all the other steps too since it's on the parent component?